guides/common/modules/proc_migrating-to-a-new-server.adoc

```diff
@@ -0,0 +1,47 @@
+[id="migrating-to-a-new-server_{context}"]
+
+= Migrating to a new server
+
+.Prerequisites
+* Install a minimal {RHEL} 8 instance on the target machine with the capacity to store backup files.
+* Do not install any {RHEL} 8 software groups or third-party applications.
+
+For more information, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_a_standard_rhel_installation/index[Performing a standard {RHEL} installation].
+
+== Performing a fresh installation of {ProjectServer} or {SmartProxyServer} to migrate to {RHEL} 8
+
+The fresh installation method has three steps:
+
+. Create a backup of the {ProjectServer} or {SmartProxyServer} on the {RHEL} 7 system source machine.
+. Perform a fresh installation of the {ProjectServer} or {SmartProxyServer} on a minimal {RHEL} 8 instance on the target machine.
+. Restore the backup on the {RHEL} 8 instance.
+
+== Performing a fresh installation of {ProjectServer} or {SmartProxyServer} on {RHEL} 8
+
+After you have created a backup of the {ProjectServer} or {SmartProxyServer} on the {RHEL} 7 source machine, you can install {ProjectServer} or {SmartProxyServer} on the {RHEL} 8 target machine.
+
+* To install {ProjectServer}, see {AdministeringDocURL}: Administering_Project/{Accessing-Server}.
+* To install {SmartProxyServer}, see {AdministeringDocURL}: Administering_Project/{installing-capsule-server}.
```

Comment:
This doesn't render correctly for me:
![image](https://user-images.githubusercontent.com/94284/174734969-290a1d0f-8e43-4a93-8d88-8f46629df9e8.png)